BUILD

```diff
@@ -125,11 +125,40 @@ PYTHON3.update({
 })
 
 NODEJS = {
+    "{REGISTRY}/{PROJECT_ID}/nodejs" + version + "-" + distro + ":" + tag_base + "-" + arch: "//nodejs:nodejs" + version + label + "_" + arch + "_" + distro
+    for arch in BASE_ARCHITECTURES
+    for distro in LANGUAGE_DISTROS
+    for version in [
+        "14",
+        "16",
+        "18",
+    ]
+    for (tag_base, label) in [
+        ("latest", ""),
+        ("debug", "_debug"),
+    ]
+}
+
+NODEJS.update({
+    "{REGISTRY}/{PROJECT_ID}/nodejs" + version + ":" + tag_base + "-" + arch: "//nodejs:nodejs" + version + label + "_" + arch + "_debian11"
+    for arch in BASE_ARCHITECTURES
+    for version in [
+        "14",
+        "16",
+        "18",
+    ]
+    for (tag_base, label) in [
+        ("latest", ""),
+        ("debug", "_debug"),
+    ]
+})
+
+NODEJS.update({
     "{REGISTRY}/{PROJECT_ID}/nodejs:14": "//nodejs:nodejs14_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs:14-debug": "//nodejs:nodejs14_debug_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:14": "//nodejs:nodejs14_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:14-debug": "//nodejs:nodejs14_debug_amd64_debian11",
```

Comment:
Don't see a point in converting this to support multi-arch for the old image naming scheme, just for a couple of months. So I think I'll just leave this be. But adding it for the new naming scheme.